Gemfile.lock

```diff
@@ -182,64 +182,62 @@ GEM
       addressable (~> 2.7)
     letter_opener (1.7.0)
       launchy (~> 2.2)
-    listen (3.4.1)
+    listen (3.5.1)
       rb-fsevent (~> 0.10, >= 0.10.3)
       rb-inotify (~> 0.9, >= 0.9.10)
-    loofah (2.9.0)
+    loofah (2.9.1)
       crass (~> 1.0.2)
       nokogiri (>= 1.5.9)
     mail (2.7.1)
       mini_mime (>= 0.1.1)
-    marcel (0.3.3)
-      mimemagic (~> 0.3.2)
+    marcel (1.0.1)
     method_source (1.0.0)
-    mimemagic (0.3.5)
-    mini_mime (1.0.2)
-    minitest (5.14.3)
+    mini_mime (1.0.3)
+    minitest (5.14.4)
     msgpack (1.4.2)
     net-scp (3.0.0)
       net-ssh (>= 2.6.5, < 7.0.0)
     net-ssh (6.1.0)
-    nio4r (2.5.5)
-    nokogiri (1.11.1-x86_64-darwin)
+    nio4r (2.5.7)
+    nokogiri (1.11.3-x86_64-darwin)
       racc (~> 1.4)
     orm_adapter (0.5.0)
-    pagy (3.10.0)
+    pagy (3.11.0)
```

Comment:
### [pagy](https://github.com/ddnexus/pagy)

**Minor** version upgrade :chart_with_upwards_trend::large_orange_diamond: 3.10.0 → 3.11.0

[_~~change-log~~, ~~source-code~~_]





<details>
<summary>Commits</summary>

A change of **5** commits. See the full changes on [the compare page](https://github.com/ddnexus/pagy/compare/3.10.0...3.11.0).

These are the individual commits:
- (2178436) [restored file removed by mistake](https://github.com/ddnexus/pagy/commit/217843624aa7e7ce114aeedfd0e4dd9cacc2d49b)
- (ebbac6b) [Fix: broken links (#272) [ci-skip]](https://github.com/ddnexus/pagy/commit/ebbac6ba2dc2f1fa8b53f9a793408cec59245f71)
- (fbc322f) [Add Swahili (sw) to pagy (#276)](https://github.com/ddnexus/pagy/commit/fbc322f6ccdcb643df1ec6eb284bffc8893ed218)
- (91e1d5d) [version 3.11.0](https://github.com/ddnexus/pagy/commit/91e1d5da727e6873d65d2e4d07e0dc787090345f)
- (b477b56) [Merge branch 'dev'](https://github.com/ddnexus/pagy/commit/b477b5633f5b5907888b0b3c2429c95b9ebea528)


</details>